src/scss/components/_form.scss

```diff
@@ -0,0 +1,85 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.form.css Form
+  *
+  * ## Form
+  * @demo demo/gaiden-css/components/form/demo.html
+**/
+
+.form {
+
+  &__container {
+    padding: {
+      right: 10%;
+      left: 10%;
+    }
+    width: 100%;
```

Comment:
@GeorgeJegon fieldset, by default, is blocked. My concern about this is a case where someone will use this `form__container` on another tag, and not in fieldset.

Maybe we can rename `form__container` to `form__fieldset`. What do you think?